lib/profiler/xprof_type.h

```diff
@@ -11,79 +11,77 @@
-    XPROF_eventRun,
-    XPROF_file_close,
-    XPROF_file_open,
-    XPROF_file_read,
-    XPROF_file_write,
-    XPROF_free,
-    XPROF_free_const,
-    XPROF_hash_lookup,
-    XPROF_headersEnd,
-    XPROF_HttpHeaderClean,
-    XPROF_HttpHeader_getCc,
-    XPROF_HttpHeaderParse,
-    XPROF_HttpMsg_httpMsgParseStep,
-    XPROF_HttpParserParseReplyLine,
-    XPROF_HttpParserParseReqLine,
-    XPROF_httpRequestFree,
-    XPROF_HttpServer_parseOneRequest,
-    XPROF_httpStart,
-    XPROF_HttpStateData_processReplyBody,
-    XPROF_HttpStateData_processReplyHeader,
-    XPROF_InvokeHandlers,
-    XPROF_malloc,
-    XPROF_MemBuf_append,
-    XPROF_MemBuf_consume,
-    XPROF_MemBuf_consumeWhitespace,
-    XPROF_MemBuf_grow,
-    XPROF_mem_hdr_write,
-    XPROF_MemObject_write,
-    XPROF_PROF_OVERHEAD,
-    XPROF_read,
-    XPROF_realloc,
-    XPROF_recv,
-    XPROF_send,
-    XPROF_SignalEngine_checkEvents,
-    XPROF_storeClient_kickReads,
-    XPROF_storeDirCallback,
-    XPROF_StoreEntry_write,
-    XPROF_storeGetMemSpace,
-    XPROF_storeMaintainSwapSpace,
-    XPROF_storeRelease,
-    XPROF_StringAllocAndFill,
-    XPROF_StringAppend,
-    XPROF_StringClean,
-    XPROF_StringInitBuf,
-    XPROF_StringReset,
-    XPROF_write,
-    XPROF_xcalloc,
-    XPROF_xmalloc,
-    XPROF_xrealloc,
-    XPROF_LAST
-} xprof_type;
+  XPROF_PROF_UNACCOUNTED,
+XPROF_aclCheckFast,
+XPROF_ACL_matches,
+XPROF_calloc,
+XPROF_clientSocketRecipient,
+XPROF_comm_accept,
+XPROF_comm_check_incoming,
+XPROF_comm_close,
+XPROF_comm_connect_addr,
+XPROF_comm_handle_ready_fd,
+XPROF_commHandleWrite,
+XPROF_comm_open,
+XPROF_comm_poll_normal,
+XPROF_comm_poll_prep_pfds,
+XPROF_comm_read_handler,
+XPROF_comm_udp_sendto,
+XPROF_comm_write_handler,
+XPROF_diskHandleRead,
+XPROF_diskHandleWrite,
+XPROF_esiExpressionEval,
+XPROF_esiParsing,
+XPROF_esiProcessing,
+XPROF_eventRun,
+XPROF_file_close,
+XPROF_file_open,
+XPROF_file_read,
+XPROF_file_write,
+XPROF_free,
+XPROF_free_const,
+XPROF_hash_lookup,
+XPROF_headersEnd,
+XPROF_HttpHeaderClean,
+XPROF_HttpHeader_getCc,
+XPROF_HttpHeaderParse,
+XPROF_HttpMsg_httpMsgParseStep,
+XPROF_HttpParserParseReplyLine,
+XPROF_HttpParserParseReqLine,
+XPROF_httpRequestFree,
+XPROF_HttpServer_parseOneRequest,
+XPROF_httpStart,
+XPROF_HttpStateData_processReplyBody,
+XPROF_HttpStateData_processReplyHeader,
+XPROF_InvokeHandlers,
+XPROF_malloc,
+XPROF_MemBuf_append,
+XPROF_MemBuf_consume,
+XPROF_MemBuf_consumeWhitespace,
+XPROF_MemBuf_grow,
+XPROF_mem_hdr_write,
+XPROF_MemObject_write,
+XPROF_PROF_OVERHEAD,
+XPROF_read,
+XPROF_realloc,
+XPROF_recv,
+XPROF_send,
+XPROF_SignalEngine_checkEvents,
+XPROF_storeClient_kickReads,
+XPROF_storeDirCallback,
+XPROF_StoreEntry_write,
+XPROF_storeGetMemSpace,
+XPROF_storeMaintainSwapSpace,
+XPROF_storeRelease,
+XPROF_StringAllocAndFill,
+XPROF_StringAppend,
+XPROF_StringClean,
+XPROF_StringInitBuf,
+XPROF_StringReset,
+XPROF_write,
+XPROF_xcalloc,
+XPROF_xmalloc,
+XPROF_xrealloc,
+  XPROF_LAST } xprof_type;
```

Comment:
What happened to this generated file formatting? I think it became worse, one several counts. Perhaps it was formatted before and now is not? This formatting change seems odd in a fully-automated commit. Is there a regression or an old problem we should fix before this goes in?

Reply:
The astyle version on master had a bump to 2.05.1 so it no longer gets run by the automation. It was being relied on to fix this up along with any dev added mistakes.

My initial checks of the new astyle are that is fixes a few things better, but also makes some files worse. I'll be working on both these in the next day or so.

Reply:
Understood. I think "format enforcement" changes should use the official astyle version. Furthermore, a switch from one official astyle version to another deserves a dedicated PR. The state of "astyle version on master [box]" does not affect the former and is dependent on the latter.

Reply:
Just to be clear this PR does *not* include any astyle changes. The version issue prevented astyle being run for this enforcement update - which is why the xprof file shows changes.

Reply:
> Just to be clear this PR does not include any astyle changes.

... which, IMO, is a problem that should be addressed before merging this PR.

> The version issue prevented astyle being run for this enforcement update

AFAICT, the "version issue" is not some insurmountable problem that has to be ignored. Instead of hiding and now ignoring it, we should fix it before this PR is merged.

FWIW, in my experience, it is fairly straightforward easy to build astyle from source if needed.